libnavigator/src/test/java/com/mapbox/navigation/navigator/LocationExTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.mapbox.navigation.navigator
+
+import android.location.Location
+import java.util.Date
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class LocationExTest {
+
+    @Test
+    fun toFixLocation() {
+        val location = Location("test").apply {
+            latitude = 11.0
+            longitude = 22.0
+            time = System.currentTimeMillis()
+            speed = 10f
+            bearing = 20f
+            altitude = 30.0
+            accuracy = 40f
+        }
+
+        val date = Date()
+        val fix = location.toFixLocation(date)
+        Assert.assertEquals(11.0, fix.coordinate.latitude(), .00001)
+        Assert.assertEquals(22.0, fix.coordinate.longitude(), .00001)
+        Assert.assertEquals(date, fix.time)
```

Comment:
I'm not personally a big fan of testing getters / setters. Of course other than this, that we should make sure that we're getting the date passed in and not the one from the `Location`.